spec/inputs/eventlog_spec.rb

```diff
@@ -1,7 +1,11 @@
 require "logstash/devutils/rspec/spec_helper"
-require 'logstash/inputs/eventlog'
 
-describe LogStash::Inputs::EventLog, :windows => true do
+describe "LogStash::Inputs::EventLog", :windows => true do
+
+  before(:all) do
+    require 'logstash/inputs/eventlog'
```

Comment:
no, no I don't ask you to move it back on top. This does the work for now. I'm wondering if there is a way to actually fix the main plugin file, to avoid having to have this issue.